#!/usr/bin/env python
PACKAGE = "rospix3"

import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("measuring", bool_t, 0, "Measuring", False)
gen.add("acquisition_duration", double_t, 1, "Acquisition duration", 0.0, 0.001, 60.0)
gen.add("measurement_duration", double_t, 2, "Measurement duration", 0.0, 0.001, 300.0)
gen.add("mask_pixels", bool_t, 4, "Mask pixels", True)

exit(gen.generate(PACKAGE, "Rospix3", "rospix3"))
